CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.13)
+
+set(FUSE_ARCHIVE_VERSION 0.0.1)
```

Comment:
Does the "0.0.1" need updating? Latest is https://github.com/google/fuse-archive/releases/tag/v0.1.9